forge-gui/res/cardsfolder/upcoming/the_cyber_controller.txt

```diff
@@ -4,7 +4,7 @@ Types:Legendary Artifact Creature Cyberman
 PT:3/3
 T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigMill1 | TriggerDescription$ When CARDNAME enters the battlefield, each opponent mills X cards. Put all creature cards milled this way onto the battlefield face down under your control. They're 2/2 Cyberman artifact creatures.
 SVar:TrigMill1:DB$ Mill | Defined$ Opponent | NumCards$ X | RememberMilled$ True | SubAbility$ DBChangeZoneAll
-SVar:DBChangeZoneAll:DB$ ChangeZone | Origin$ Graveyard,Exile | GainControl$ True | Destination$ Battlefield | Defined$ Remembered | FaceDown$ True | FaceDownPower$ 2 | FaceDownToughness$ 2 | FaceDownSetType$ Artifact & Creature & Cyberman | SubAbility$ DBCleanup
```

Comment:
@tool4ever ChangeZone + `Defined$ Remembered.Creature` better ?

Reply:
Sure, I think this was already attempted here but lost the type in the refactor
@Simisays
you understand the difference?